spec/fabricators/category_fabricator.rb

```diff
@@ -0,0 +1,3 @@
+Fabricator(:category) do
+  name { Faker::Lorem.words(1) }
```

Comment:
Actually this is not right, `Faker::Lorem.words(1)` produces an array of words - 1. And name is a string. It should be then turned to a string with `#join` (check Ruby docs for Array#join` or just use `word` instead of `words(1)`.